config/distro_signatures.json

```diff
@@ -273,6 +289,38 @@
     "kernel_options_post":"",
     "boot_files":[]
    },
+   "fedora29": {
+    "signatures":["Packages"],
+    "version_file":"(fedora)-release-29-(.*)\\.noarch\\.rpm",
+    "version_file_regex":null,
+    "kernel_arch":"kernel-(.*)\\.rpm",
+    "kernel_arch_regex":null,
+    "supported_arches":["aarch64","i386","x86_64","ppc64le"],
+    "supported_repo_breeds":["rsync", "rhn", "yum"],
+    "kernel_file":"vmlinuz(.*)",
+    "initrd_file":"initrd(.*)\\.img",
+    "isolinux_ok":false,
+    "default_kickstart":"/var/lib/cobbler/kickstarts/sample_end.ks",
+    "kernel_options":"repo=$tree",
+    "kernel_options_post":"",
+    "boot_files":[]
+   },
+   "fedora30": {
```

Comment:
You opened a seperate PR for this one. Was it intended that you doubled the PR or was this a mistake?

Reply:
This PR is dedicated to the release28 branch over the master branch (cobbler 3.0)

Reply:
thx for the review.

Reply:
Oh I should have taken a closer look of couse then this is even more lgtm.